### ISSUE TITLE
Plex buffers then playback error after 15 mins...login error?

### ISSUE THREAD

**Issue by xXTRIPPA420Xx** [tgorgdotcom/locast2plex#110]
Server Version#:1.20.2.3402
Player Version#: 5.2.1
Tuner Make/Model: locast2plex as windows service
Guide/Lineup name: Boston
Using XMLTV?: ?
Channel number/Name: All

Recently installed locast2plex as a windows service. After exactly 15 minutes of playback I get buffering then a playback error. This happens every time after exactly 15 minutes. I’m thinking it could be a login error with locast but i have my credentials correct (if they are not correct in config file windows service will not start). Can anyone offer any help on this? When creating the service do i enter my locast login info in arguments or does it grab login from the config file?

Here is what I filled in to the GUI
path: C:\Python27\Python27.exe
Startup directory: C:\Python27

Arguments: C:\code\locast2plex\main.py -u:usernamexxx -p:passwordxxx --debug --addy:127.0.0.1 --port:6077
Service name is locast2plex

Click Install Service.

**Comment by larry5601** [comment]
Did you donate to locast? This sound like how often they stop playing TV and give you the commercial to donate.

When I went to the version with the config file all the parameters were in the config file. The command line became
C:\code\locast2plex\main.py

**Comment by xXTRIPPA420Xx** [comment]
Yes, ive donated to locast. I changed the command line to just C:\code\locast2plex\main.py and recreated the service and i still get buffering after 15 minutes. I can see in command prompt locast2plex does log in and says i donate. i figured out this is only happening on my xbox. playback on roku or web works fine.

**Comment by deathbybandaid** [comment]
Swap the commandline arguments for config file
